src/streaming/controllers/MediaController.js

```diff
@@ -100,10 +94,10 @@ function MediaController() {
         }
 
         if (tracks.length === 0) {
-            setTrack(selectInitialTrack(tracksForType));
+            setTrack(type === Constants.FRAGMENTED_TEXT ? tracksForType[0] : selectInitialTrack(tracksForType), true);
```

Comment:
Suggestion: Move this logic to selectInititalTrack as well. In this function we can check for the track type and return the first track for FRAGMENTED_TEXT by default

Reply:
done